src/Microsoft.ML.TensorFlow/TensorflowUtils.cs

```diff
@@ -90,7 +90,16 @@ internal static DataViewSchema GetModelSchema(IExceptionContext ectx, Graph grap
                     DataViewType columnType = new VectorDataViewType(mlType);
                     if (!(Utils.Size(tensorShape) == 1 && tensorShape[0] <= 0) &&
                         (Utils.Size(tensorShape) > 0 && tensorShape.Skip(1).All(x => x > 0)))
-                        columnType = new VectorDataViewType(mlType, tensorShape[0] > 0 ? tensorShape : tensorShape.Skip(1).ToArray());
+                        if (treatOutputAsBatched)
```

Comment:
It'd be good if you could add another more detailed explanation for this if/else here.

Reply:
Added more documentation to both the if and else.